pep-0484.txt

```diff
@@ -600,6 +600,17 @@ the type (class) of the objects created by instantiating them doesn't
 record the distinction.  This behavior is called "type erasure"; it is
 common practice in languages with generics (e.g. Java, TypeScript).
```

Comment:
The previous paragraphs suggest that you have to create a type alias (e.g. IntNode) but that's no longer necessary (since it's no longer as slow). Maybe you can update that while you're at it? I'd still recommend using a type alias (since it's clearer and still a tiny bit faster) but it's not strictly necessary.